PWGHF/vertexingHF/vHFML/AliAnalysisTaskSEDs.h

```diff
@@ -212,12 +222,13 @@ class AliAnalysisTaskSEDs : public AliAnalysisTaskSE
   Bool_t fKeepOnlyBkgFromHIJING = kFALSE;       /// flag to keep the background from HIJING only
   /// variables for ML application
   Bool_t fApplyML = kFALSE;                     /// flag to enable ML application
+  Bool_t fMultiClassML = kFALSE;                /// flag to enable multi-class models (Bkg, Prompt, FD)
   Bool_t fUseMinimalVarForSparse = kFALSE;      /// flag to keep only inv. mass, pt and prob. in the sparse
   TString fConfigPath = "";                     /// path to ML config file
   AliHFMLResponseDstoKKpi* fMLResponse = nullptr;  //!<! object to handle ML response
-  Int_t fNMLBins = 300;                         /// number of bins for ML output axis in THnSparse
-  Double_t fMLOutputMin = 0.85;                 /// min for ML output axis in THnSparse
-  Double_t fMLOutputMax = 1.0;                  /// max for ML output axis in THnSparse
+  Int_t fNMLBins[3] = {100,100, 100};           /// number of bins for ML output axis in THnSparse
```

Comment:
Since you modified these data members you should also update the `ClassDef` from 41 to 42